modules/src/ics03_connection/msgs.rs

```diff
@@ -25,40 +25,20 @@ pub mod conn_open_try;
 pub const TYPE_MSG_CONNECTION_OPEN_CONFIRM: &str = "connection_open_confirm";
 
 /// Enumeration of all possible messages that the ICS3 protocol processes.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub enum ConnectionMsg {
     ConnectionOpenInit(MsgConnectionOpenInit),
     ConnectionOpenTry(Box<MsgConnectionOpenTry>),
-    ConnectionOpenAck(MsgConnectionOpenAck),
+    ConnectionOpenAck(Box<MsgConnectionOpenAck>),
```

Comment:
Yes, forgot to mention, `Ack` is also too large now because tendermint client state (included in the message) has new fields (e.g.consensus params, proofOps). And we don't allow `large_enum_variant` for `ConnectionMsg`. It would be better imo to allow it and to remove `Box` from `Ack` and `Try` as they make 50% of all variants. Let me know what you think.

Reply:
Ah I suspected some field must have grown in the meantime, but I missed which one exactly. Thanks for the explanation!

I have no particular opinion about introducing a `allow(clippy::..)` here. In the interest of merging this soon, maybe we keep things as they are currently. In future, I think this deserves a discussion about clippy in general, if we want to have a repo-wide configuration where we adjust the clippy parameters (suggested in [another comment](https://github.com/informalsystems/ibc-rs/pull/355#discussion_r516654739)) so we can remove the allow/clippy annotations altogether.